README.md

```diff
@@ -91,7 +117,7 @@ DESCRIPTION
   If no version is specified, the head version will be returned.
 ```
 
-_See code: [src/commands/get/build.js](https://github.com/warehouseai/wrhs/blob/v0.3.1/src/commands/get/build.js)_
```

Comment:
the v0.3.1 links were all broken

Reply:
These are generated by oclif. Looks like we tag as `0.3.1` and they are expecting `v0.3.1`. It looks like there is some support for configuring this, but I would have to dig more. We might want to just tag this repo with the `v`.
https://github.com/oclif/dev-cli/blob/v1.22.1/src/commands/readme.ts#L171